apps/src/lib/levelbuilder/unit-editor/UnitEditor.jsx

```diff
@@ -911,21 +904,6 @@ class UnitEditor extends React.Component {
               this.setState({projectWidgetTypes})
             }
           />
-          {!this.props.isMigrated && (
```

Comment:
I probably should have started deleting from this point in the code, since everything on levelbuilder contingent upon `!isMigrated` is dead 🤷